Add `FAQ` page (#14)
osuTitanic/wiki: wiki/FAQ/en.md

# FAQ

This is a list of common questions that get asked.

## What is this?

> Titanic! is a private server, that was designed to support osu! stable clients from 2008 to 2014 and only maps from that era (With a few exceptions). It's a community-driven project that is fully [open-source](https://github.com/osuTitanic), so feel free to look at the code and contribute to it.

Reviewer: ```suggestion
Titanic! is a private server for [osu!](https://osu.ppy.sh), that was designed to preserve the earlier days of the game, by letting players use clients from 2008 to 2014, as well as similar maps from that era. It's a community-driven project that is fully [open-source](https://github.com/osuTitanic), so feel free to look at the code and contribute to it.
```

## How can I play?

To get started you can take a look at the [[installation guide]] or just go to the [downloads page](https://osu.titanic.sh/download/) on our website.

> ## How can I get beatmaps.

Reviewer: ```suggestion
## How do I find beatmaps?
```

You can download beatmaps via osu! Direct or on the [beatmaps page](https://osu.titanic.sh/beatmapsets/) on the website. Downloading maps from the offical osu! website works as well.

## My game is feeling slow/laggy, how can I fix that?

> Many players sadly face this issue and most often it can be fixed by setting the RefreshRate value in the osu!.yourname.cfg file, to your monitors refresh rate. If you are still experiencing issues, please try to mess around with the graphics & fps settings, until it feels right!

Reviewer: ```suggestion
Many players sadly face this issue and most often it can be fixed by setting the `RefreshRate` value in the `osu!.yourname.cfg` file, to your monitors refresh rate. If you are still experiencing issues, please try to mess around with the graphics & fps settings, until it feels right! Other versions of the game might also give you a better or worse experience.
```

## Will RX/AP scores be submitted?

> Yes we support Relax and AutoPilot submittion. We currently do not have seperate leaderboards for these at the moment. They do give PP and score but not as much compared to vanillia to keep things fair.

Reviewer: ```suggestion
Yes, we do support Relax and Autopilot submission, with one noticeable difference to other servers: Ranks for Vanilla, Relax and Autopilot are merged together, since they do give PP and score, but not as much compared to Vanilla to keep things fair.
```

## What PP system is this?

We use a customized pp system, mostly developed by [Adachi](https://osu.titanic.sh/u/39), which changes the behavior of relax & autopilot PP and also has some changes regarding combo scaling, which makes a few misses not kill your pp. 
We also have a ["ppv1" recreation](https://osu.titanic.sh/rankings/osu/ppv1), which was meant to be a joke, but seems to be a real thing now. Please keep in mind that this is nowhere near the original ppv1 system!

## How can I submit a beatmap?

You have two options for this. Either submit one from the editor in-game, or request a map from osu.ppy.sh on the [map requests](https://osu.titanic.sh/forum/11) page on the fourms.

## Where can I report bugs or suggest features?

> You can suggest features in the the [Feature Requests](https://osu.titanic.sh/forum/6) page on the forums.

Reviewer 2: ```suggestion
You can suggest features in the [Feature Requests](https://osu.titanic.sh/forum/6) page on the forums.
```

> You have two options for this. Either submit one from the editor in-game, or request a map from osu.ppy.sh on the [map requests](https://osu.titanic.sh/forum/11) page on the fourms.

Reviewer 2: ```suggestion
There's two options for this. Either submit one from the editor in-game, or request a map from osu.ppy.sh on the [map requests](https://osu.titanic.sh/forum/11) page on the forums.
```
